.github/workflows/e2e-browser.yml

```diff
@@ -39,10 +39,14 @@ jobs:
         if: ${{ github.actor != 'dependabot[bot]' }}
         run: npm run test:e2e:browser
         env:
-          E2E_TEST_IDP: ${{ secrets.E2E_TEST_IDP }}
-          E2E_TEST_ENVIRONMENT: ${{ matrix.environment-name }}
           E2E_TEST_USER: ${{ secrets.E2E_TEST_USER }}
           E2E_TEST_PASSWORD: ${{ secrets.E2E_TEST_PASSWORD }}
+          E2E_TEST_IDP: ${{ secrets.E2E_TEST_IDP }}
+          E2E_TEST_OWNER_CLIENT_ID: ${{ secrets.E2E_TEST_OWNER_CLIENT_ID }}
+          E2E_TEST_OWNER_CLIENT_SECRET: ${{ secrets.E2E_TEST_OWNER_CLIENT_SECRET }}
+          E2E_TEST_REQUESTOR_CLIENT_ID: ${{ secrets.E2E_TEST_REQUESTOR_CLIENT_ID }}
+          E2E_TEST_REQUESTOR_CLIENT_SECRET: ${{ secrets.E2E_TEST_REQUESTOR_CLIENT_SECRET }}
```

Comment:
If that feels useful, you may add a comment here mentioning these secrets are used for test setup, as it may come as a surprise when looking at browser-based tests.